syncer/dml.go

```diff
@@ -284,17 +284,16 @@ func genDeleteSQL(schema string, table string, value []interface{}, columns []*c
 }
 
 func genColumnList(columns []*column) string {
-	var columnList []byte
+	var buf strings.Builder
 	for i, column := range columns {
-		name := fmt.Sprintf("`%s`", column.name)
-		columnList = append(columnList, []byte(name)...)
-
 		if i != len(columns)-1 {
-			columnList = append(columnList, ',')
+			fmt.Fprintf(&buf, "`%s`,", column.name)
+		} else {
+			fmt.Fprintf(&buf, "`%s`", column.name)
```

Comment:
Using `&buf` would make the string builder escape to heap.

It would be faster if we do 

```go
buf.WriteString("`" + column.name + "`")
```

Proof: https://play.golang.org/p/Pjl36V-QGho
Analysis: `go test -gcflags="-m" -v --bench . escape_test.go`